import/controls/permissionmanager.cpp

```diff
@@ -0,0 +1,52 @@
+#include "permissionmanager.h"
```

Comment:
I've just noticed that this file, and import/controls/permissionmanager.h still need copyright headers.